lib/common/output_html.c

```diff
@@ -256,9 +256,9 @@ html_err(pcmk__output_t *out, const char *format, ...) {
 }
 
 G_GNUC_PRINTF(2, 3)
-static void
+static int
 html_info(pcmk__output_t *out, const char *format, ...) {
-    /* This function intentially left blank */
+    return pcmk_rc_ok;
```

Comment:
Should html/none/xml return no output?

Reply:
I could see it going either way.  I chose pcmk_rc_ok because they did what they were supposed to, but if we want to strictly interpret the return code, pcmk_rc_no_output would be more correct.

Reply:
Updated to have those versions return pcmk_rc_no_output.